Sources/PrimerSDK/Classes/Services/API/Primer/PrimerAPI.swift

```diff
@@ -28,7 +28,7 @@ enum PrimerAPI: Endpoint {
 extension PrimerAPI {
 
     // MARK: Base URL
-    var baseURL: String {
+    var baseURL: String? {
```

Comment:
Nice fix! Usually the `baseURL` is hardcoded, so you should have the `fatalError`. However, on our case the backend returns the URL, so we shouldn't crash if we don't receive it.